src/main/kotlin/app/revanced/patches/grindr/unlimited/patch/setUnlimited.kt

```diff
@@ -0,0 +1,125 @@
+package app.revanced.patches.grindr.unlimited.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.extensions.exception
+
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patches.grindr.unlimited.annotations.UnlockUnlimitedCompatibility
+import app.revanced.patches.grindr.unlimited.fingerprints.*
+import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
+import app.revanced.patcher.extensions.InstructionExtensions.replaceInstructions
+import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
+import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
+
+import app.revanced.patches.grindr.patch.bytecode.FirebaseGetCertPatch
+
+@Patch
+@Name("Unlock unlimited")
+@Description("Unlocks unlimited features.")
+@UnlockUnlimitedCompatibility
+@DependsOn([FirebaseGetCertPatch::class])
+class UnlockUnlimitedPatch : BytecodePatch(
+    listOf(
+        HasFeatureFingerprint,
+        IsFreeFingerprint,
+        IsNoPlusUpsellFingerprint,
+        IsNoXtraUpsellFingerprint,
+        IsPlusFingerprint,
+        IsUnlimitedFingerprint,
+        IsXtraFingerprint,
+        InnaccessibleProfileManagerbFingerprint,
+        InnaccessibleProfileManagerdFingerprint
```

Comment:
Are these many fingerprints necessary? Is there no single location to toggle all the unlimited features?